src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs

```diff
@@ -16,23 +16,25 @@ namespace NHSD.BuyingCatalogue.Identity.Api.Services
 {
     public sealed class ProfileService : IProfileService
     {
-        private readonly IUserRepository _applicationUserRepository;
+        private readonly IUsersRepository _applicationUserRepository;
 
-        private static readonly IDictionary<string, IEnumerable<Claim>> _organisationFunctionClaims =
-            new Dictionary<string, IEnumerable<Claim>>
+        private static readonly IDictionary<OrganisationFunction, IEnumerable<Claim>> _organisationFunctionClaims =
+            new Dictionary<OrganisationFunction, IEnumerable<Claim>>
             {
-                { 
-                    "Authority", new List<Claim>
+                {
+                    OrganisationFunction.Authority,
+                    new List<Claim>
                     {
                         new Claim(ApplicationClaimTypes.Organisation, ApplicationPermissions.Manage),
                         new Claim(ApplicationClaimTypes.Account, ApplicationPermissions.Manage)
                     }
                 }
             };
 
-        public ProfileService(IUserRepository applicationUserRepository)
+        public ProfileService(IUsersRepository applicationUserRepository)
         {
-            _applicationUserRepository = applicationUserRepository ?? throw new ArgumentNullException(nameof(applicationUserRepository));
+            _applicationUserRepository = applicationUserRepository ??
+                                         throw new ArgumentNullException(nameof(applicationUserRepository));
```

Comment:
This one boils down to personal preference, so its up to you how you leave it, but In this case I think that it would look better without the line wrap, thoughts?

Reply:
Prefer not to horizontal scroll if I can help it, probably get away with it on this one. Just thought it looked neater on a newline.